docs/.vitepress/components/Article.vue

```diff
@@ -1,20 +1,21 @@
 <script setup lang="ts">
 defineProps<{
-	title: string;
+	title?: string;
```

Comment:
Are there really cases when the title is undefined? If so we probably need to handle it, e.g. hiding the header

Reply:
I’m literally doing it on the homepage for the framework guides. But now I look at it, I suppose they could just be images